packages/expo-router/src/native-tabs/NativeBottomTabs/native-tabs-styles.web.ts

```diff
@@ -0,0 +1,3 @@
+import styles from '../../../assets/native-tabs.module.css';
```

Comment:
You can just import module.css on native and it will no-op like this in the bundler.